platforms/cpuinfo_other.go

```diff
@@ -0,0 +1,59 @@
+//go:build windows || darwin || freebsd
+// +build windows darwin freebsd
+
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package platforms
+
+import (
+	"runtime"
+)
+
+func getCPUVariant() string {
+
+	var variant string
+
+	if runtime.GOOS == "windows" || runtime.GOOS == "darwin" {
```

Comment:
Since we are checking GOOS here, how about building this file on `!linux`? We right now have to sync this `if` condition, the build flags on this file and cpuVariant's `if` condition.

Reply:
That makes sense. I updated to !linux.